### PR TITLE
[WebAPI] Remove unused WebGL test cases

### PR DESCRIPTION
**Pull request by renchenglei**
BUG=https://crosswalk-project.org/jira/browse/CTS-1551

### CODE REVIEW

**Comment by renchenglei** [comment]
sync with @BruceDai  and @haoxli , we remove unreferenced files to enable all webgel test cases run smoothly on WTS.
@zqzhang ptal, thanks a lot~

File: webapi/webapi-webgl-khronos-tests/webgl/khronos/COPYING [hunk 1]
```diff
@@ -93,6 +93,63 @@ function testPassed(assertName, name) {
  }
 
 
+Remove unused WebGL test cases:
```
Comment (zqzhang) [inline webapi/webapi-webgl-khronos-tests/webgl/khronos/COPYING]: Just curious why these tests are unused? Are they used in Khronos conformance test suite? If not, why Khronos keeps them?

Comment (renchenglei) [inline webapi/webapi-webgl-khronos-tests/webgl/khronos/COPYING]: @zqzhang , These removed tests are not included in [webgl-conformance-tests](https://www.khronos.org/registry/webgl/conformance-suites/1.0.3/webgl-conformance-tests.html). Some of them may cause browser crash or run a long time, the others are performance tests. So, the Khronos conformance test suite doesn't include these tests.

**Comment by zqzhang** [comment]
Per offline talk to @renchenglei , in future if Khronos runs these tests in its testing, we will add them back. At current being, let's remove these test files to get consistent test results between regular packaged test suite and web testing service. Thanks.

**Pull request merged**